exchange_app/api_1_0/address.py

```diff
@@ -1,11 +1,35 @@
+import re
+from hashlib import sha256
 from flask import jsonify
 from . import api
 
 
+def __decode_base58(bc, length):
```

Comment:
Why use of __ prefix ? No special meaning in modules , only for classes @stdevOzkar cmiiw .